.github/workflows/canary-integration-test.yml

```diff
@@ -21,883 +21,890 @@ jobs:
+          # wait until osd.1 is removed from the osd tree
+          timeout 120 sh -c "while kubectl -n rook-ceph exec $toolbox -- ceph osd tree|grep -qE 'osd.1'; do echo 'waiting for ceph osd 1 to be purged'; sleep 1; done"
+          kubectl -n rook-ceph exec $toolbox -- ceph status
+          kubectl -n rook-ceph exec $toolbox -- ceph osd tree
+
+      - name: collect common logs
+        if: always()
+        uses: ./.github/workflows/collect-logs
+        with:
+          name: canary
+
+      - name: setup tmate session for debugging when event is PR
+        if: failure() && github.event_name == 'pull_request'
+        uses: mxschmitt/action-tmate@v3
+        timeout-minutes: 60
 
   pvc:
     runs-on: ubuntu-18.04
     if: "!contains(github.event.pull_request.labels.*.name, 'skip-ci')"
     steps:
-    - name: checkout
-      uses: actions/checkout@v2
-      with:
-        fetch-depth: 0
-
-    - name: setup cluster resources
-      uses: ./.github/workflows/setup-cluster-resources
-      with:
-        github-token: ${{ secrets.GITHUB_TOKEN }}
-
-    - name: use local disk and create partitions for osds
-      run: |
-        tests/scripts/github-action-helper.sh use_local_disk
-        tests/scripts/github-action-helper.sh create_partitions_for_osds
-
-    - name: create cluster prerequisites
-      run: |
-        BLOCK=$(sudo lsblk --paths|awk '/14G/ {print $1}'| head -1)
-        tests/scripts/localPathPV.sh "$BLOCK"
-        tests/scripts/github-action-helper.sh create_cluster_prerequisites
-
-    - name: deploy cluster
-      run: |
-        tests/scripts/github-action-helper.sh deploy_manifest_with_local_build deploy/examples/operator.yaml
-        yq write -i tests/manifests/test-cluster-on-pvc-encrypted.yaml "spec.storage.storageClassDeviceSets[0].encrypted" false
-        yq write -i tests/manifests/test-cluster-on-pvc-encrypted.yaml "spec.storage.storageClassDeviceSets[0].count" 2
-        yq write -i tests/manifests/test-cluster-on-pvc-encrypted.yaml "spec.storage.storageClassDeviceSets[0].volumeClaimTemplates[0].spec.resources.requests.storage" 6Gi
-        kubectl create -f tests/manifests/test-cluster-on-pvc-encrypted.yaml
-        tests/scripts/github-action-helper.sh deploy_manifest_with_local_build deploy/examples/toolbox.yaml
-
-    - name: wait for prepare pod
-      run: tests/scripts/github-action-helper.sh wait_for_prepare_pod
-
-    - name: wait for ceph to be ready
-      run: tests/scripts/github-action-helper.sh wait_for_ceph_to_be_ready osd 2
-
-    - name: check-ownerreferences
-      run: tests/scripts/github-action-helper.sh check_ownerreferences
-
-    - name: collect common logs
-      if: always()
-      uses: ./.github/workflows/collect-logs
-      with:
-        name: pvc
-
-    - name: setup tmate session for debugging when event is PR
-      if: failure() && github.event_name == 'pull_request'
-      uses: mxschmitt/action-tmate@v3
-      timeout-minutes: 60
+      - name: checkout
```

Comment:
Can you adjust your editor's auto-indentation so it doesn't change all these blocks? It's difficult to see the diff, and we don't want it changing all the time between commits.

Reply:
My appologies,  I didn't check the latest push, Editor messed it up. fixing it.